dragonfly/block/cocoa_bean.go

```diff
@@ -0,0 +1,122 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/dragonfly/block/model"
+	"github.com/df-mc/dragonfly/dragonfly/block/wood"
+	"github.com/df-mc/dragonfly/dragonfly/item"
+	"github.com/df-mc/dragonfly/dragonfly/item/tool"
+	"github.com/df-mc/dragonfly/dragonfly/world"
+	"github.com/go-gl/mathgl/mgl64"
+	"math/rand"
+)
+
+// CocoaBean is a crop block found in Jungle biomes.
+type CocoaBean struct {
+	noNBT
+
+	// Facing is the direction from the cocoa bean to the log.
+	Facing world.Direction
+	// Age is the stage of the cocoa bean's growth. 2 is fully grown.
```

Comment:
This says 2 is fully grown, but [this line](https://github.com/df-mc/dragonfly/pull/52/files#diff-5612e1f1f31a54009b740832a3950540R76) checks for age 3.